docs/pages/guides/cache-builds-remotely.mdx

```diff
@@ -74,9 +72,9 @@ type CalculateFingerprintHashProps = {
 };
 ```
 
-A reference implementation using GitHub Releases to cache builds can be found at in the [Remote Build Cache Provider Example](https://github.com/expo/examples/tree/master/with-github-remote-build-cache-provider)
+A reference implementation using GitHub Releases to cache builds can be found at in the [Build Cache Provider Example](https://github.com/expo/examples/tree/master/with-github-remote-build-cache-provider)
```

Comment:
```suggestion
A reference implementation using GitHub Releases to cache builds can be found in the [Build Cache Provider Example](https://github.com/expo/examples/tree/master/with-github-remote-build-cache-provider).
```

Unrelated to the changes in this PR but let's improve the verbiage here and also add the missing period at the end of the sentence.